docs/README-develop-beaker-scripts.md

```diff
@@ -121,7 +60,7 @@ HOSTS:
     <Nexus agent>:
         roles:
             - agent
-        platform: cisco-5-x86_64
+        platform: cisco_nexus-7-x86_64
```

Comment:
So... what is the relevance of `-6-` and `-7-` ? I'm not suggesting you change this, just curious when / if this would change to `-8-` ?